themes/custom/ts_wrin/sass/style.scss

```diff
@@ -18,18 +18,8 @@
 @import "global/tabs.scss";
 @import "global/tray-nav.scss";
 @import "global/ts-hamburger-menu.scss";
-.main-content,
-.main-content .text-formatted,
-.paragraph--type--side-by-side .text-formatted {
-  line-height: 1.5;
+@import "global/wysiwyg.scss";
```

Comment:
I'm going to revert this and do it a slightly different way -- I pull in the WYSIWYG file this way so ckeditor can also preview our text styles. (See that it's also pulled in `web/profiles/contrib/wri_sites/themes/custom/ts_wrin/sass/ckeditor.scss` too). So wrapping our contents with these classes makes that not work anymore. I'm just pulling the page-specific styles into their own component file.